src/test/java/test/HelmOutputComparisonTest.java

```diff
@@ -47,6 +47,7 @@ private void captureHelmTemplateOutput(String product, Path outputFile) throws E
                         getHelmReleaseName(product),
                         getHelmChartPath(product).toString(),
                         "--debug",
+                        "-n mynamespace",
```

Comment:
I was getting different test results depending on which environment I was running in, so I added this to make it predictable